### PR TITLE
Add initial minimal ArmNN delegate plugin.

### PR DESCRIPTION
**Pull request by GeorgeARM**
- Adds ArmNN delegate interface
- Allows to specify backend id
- Supports the following operators: Conv2d, DepthwiseCond2d, Pool2d, Softmax, Squeeze

### CODE REVIEW

**Comment by petewarden** [comment]
@jdduke could you take a look at this, since it's in the mobile TensorFlow Lite code?

**Comment by jdduke** [comment]
Thanks George, per offline discussion we'll sync up on this directly to discuss next steps.

**Comment by gbaned** [comment]
@GeorgeARM, @jdduke Any update on this PR, please. Thanks!

**Comment by jdduke** [comment]
Tentatively closing per offline discussions, we can reopen after further dialog around hosting and distribution.
